src/snowflake/snowpark/modin/plugin/_internal/aggregation_utils.py

```diff
@@ -270,6 +270,29 @@ def _columns_coalescing_idxmax_idxmin_helper(
     "quantile": column_quantile,
     "nunique": count_distinct,
 }
+GROUPBY_AGG_PRESERVES_SNOWPARK_PANDAS_TYPE = (
```

Comment:
can you add comment for `SNOWFLAKE_BUILTIN_AGG_FUNC_MAP` to notice if any change is made in that map, please consider update these two maps too?